base/floatfuncs.jl

```diff
@@ -274,6 +274,15 @@ function isapprox(x::Number, y::Number; atol::Real=0, rtol::Real=rtoldefault(x,y
     x == y || (isfinite(x) && isfinite(y) && abs(x-y) <= max(atol, rtol*max(abs(x), abs(y)))) || (nans && isnan(x) && isnan(y))
 end
 
+"""
+    isapprox(x; kwargs...) / ≈(x; kwargs...)
+
+Create a function that compares its argument to `x` using `≈`, i.e. a function equivalent to `y -> y ≈ x`.
+
+The keyword arguments supported here are the same as those in [`isapprox`](@ref).
+"""
+isapprox(y; kwargs...) = x -> isapprox(x, y; kwargs...)
```

Comment:
I think all of these changes can/should instead go in `Base/operators.jl` and use the `Fx2` machinery?

See https://github.com/JuliaLang/julia/blob/90d481e6ca1f50758841d980d0810967ccc627f6/base/operators.jl#L930-L939

Reply:
likewise, test could then go here? https://github.com/JuliaLang/julia/blob/90d481e6ca1f50758841d980d0810967ccc627f6/test/operators.jl#L200

Reply:
Unfortunately, I don't have enough time to push this through today.  Will get to it tomorrow, though.

The reason I didn't put it in `operators` was because `isapprox` and `≈` were defined in `floatfuncs`.

Reply:
Bump? I was about to make exactly this PR, but then I found yours! Would be nice to have.

Reply:
The problem with using `Fix2` is that it does not support capturing keyword arguments.